nextbike/io/output.py

```diff
@@ -3,14 +3,17 @@
 import pickle
 
 
-def save_model(model):
-    pickle.dump(model, open(os.path.join(get_data_path(), "output/model.pkl"), 'wb'))
+def save_object(pObject, pFilename):
+    pickle.dump(pObject, open(os.path.join(get_data_path(), "output", "models", pFilename), 'wb'))
+    print("Saved", pFilename)
 
 
+# TODO: No need for save_fig and saveTrip => Create just one method
```

Comment:
Leider doch - beim einen rufen wir df.to_csv, beim anderen fig.savefig

Würde ich einfachheitshalber dabei belassen :)

Reply:
Aber eine allgemeine save_df statt der saveTrip fänd ich gut 👍

Reply:
we could create one method which receives the type as which it should be saved

Reply:
i mean it's not the file ending but the object calling the save method which differs..
so i would recommend to stay to two methods - one for figures and one for df's which gets a filename as parameter :)

Reply:
an if would be needed therefore...
But yes you are right. Two methods would be more readable.

Reply:
i think it's fine like it is now :)